docs/src/rules/class-methods-use-this.md

```diff
@@ -6,55 +6,92 @@ further_reading:
 - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static
 ---
 
+[Classes](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes) are often used to encapsulate reusable logic, especially stateful logic, into an object, where each instance's state is accessed via `this`. When an API is written with an instance method, it signals to consumers:
 
-If a class method does not use `this`, it can *sometimes* be made into a static function. If you do convert the method into a static function, instances of the class that call that particular method have to be converted to a static call as well (`MyClass.callStaticMethod()`)
+- The method's outcome is related to the object on which it's invoked, including possibly its state.
 
-It's possible to have a class method which doesn't use `this`, such as:
+	```js
+	const array1 = [1, 2, 3];
+	const array2 = [4, 5, 6];
 
-```js
-class A {
-    constructor() {
-        this.a = "hi";
-    }
+    // Using the `includes()` method on different objects gives different results:
+	array1.includes(1); // true
+	array2.includes(1); // false
 
-    print() {
-        console.log(this.a);
-    }
+	// Modifying the state of an object may change the outcome of its instance methods:
+	array2.push(1);
+	array2.includes(1); // true
+	```
+
+- The method doesn't make sense to be used without an associated object. (For example, it doesn't make sense to call `Array#includes()` without an array to operate on.)
 
+If a class instance method does not use `this`, that normally means that it does not access any information or state about its instance, meaning its outcome is not tied to the object on which it is invoked, and it might even make sense to invoke without any object at all.
+Therefore, it can *sometimes* be refactored into an [ordinary function](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions) or a [static method](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/static), which may better communicate intent to users of the API.
+Be aware that if you do convert the method into a function, this will require _all_ existing usages of the instance method to update to the new API, which is often a breaking change.
```

Comment:
I'm not sure what this paragraph is meant to communicate. Converting a method into a function means necessarily means refactoring, which to me goes without saying. I'm also not sure why this might be a breaking change. Breaking changes are only relevant for public-facing APIs and I don't think we should assume that's what the user is doing.

I think we can probably just delete this paragraph.

Reply:
I'd say let's just delete it. I get your point, I just think it's too nuanced to include in the docs (that most people just lightly skim).